src/fundus/publishers/fr/__init__.py

```diff
@@ -13,3 +14,14 @@ class FR(PublisherEnum):
         ],
         parser=LeMondeParser,
     )
+
+    LeFigaro = PublisherSpec(
+        name="Le Figaro",
+        domain="https://www.lefigaro.fr/",
+        sources=[
+            Sitemap("https://sitemaps.lefigaro.fr/lefigaro.fr/articles.xml"),
+            Sitemap("https://www.lefigaro.fr/elections/resultats/sitemap.xml"),
```

Comment:
You can remove this `https://www.lefigaro.fr/elections/resultats/sitemap.xml` sitemap.